source/_docs/ecosystem/nginx.markdown

```diff
@@ -111,6 +111,7 @@ http {
 
         # These shouldn't need to be changed
         listen 443 default_server;
+        listen [::]:443 default_server ipv6only=on;
```

Comment:
I would argue that the current solution is the same as the port handling of port 80 (see above that line in the same file).
Binding to v6 individually works on every version, so I do not really see a reason to change the PR.